backend/import/mo_api.py

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import os
+
+from collections import defaultdict
+
+import requests
+
+from cached_property import cached_property
+
+
+DEFAULT_MO_URL = os.environ.get(
+        'MO_URL', 'http://morademo.atlas.magenta.dk/service'
+)
+ORG_ROOT = os.environ.get(
+    'MO_ORG_ROOT', '293089ba-a1d7-4fff-a9d0-79bd8bab4e5b'
+)
+
+
+def mo_get(url):
```

Comment:
You should move this to an instance method on `MOData`, and stash an instance of `requests.Session` on the object itself. Sessions can provide a noticeable speedup through connection pooling and HTTP pipelining.

Reply:
The implementation is [here](https://github.com/requests/requests/blob/v2.20.1/requests/api.py#L59): 

```python
def request(method, url, **kwargs):
[snip]
    # By using the 'with' statement we are sure the session is closed, thus we
    # avoid leaving sockets open which can trigger a ResourceWarning in some
    # cases, and look like a memory leak in others.
    with sessions.Session() as session:
        return session.request(method=method, url=url, **kwargs)
```

The documentation is [here](http://docs.python-requests.org/en/master/user/advanced/).

Reply:
I will look into that! If it's a instance method on MOData, we'll always have one session per thread in the current implementation, which should be fine.